apps/state/lib/state/connecting_stops.ex

```diff
@@ -31,7 +31,9 @@ defmodule State.ConnectingStops do
     "place-DB-2205" => %{add: ~w(16391)},
     "place-FR-0064" => %{add: ~w(2137)},
     "place-GRB-0118" => %{add: ~w(3806)},
-    "place-harsq" => %{add: ~w(110)}
+    "place-harsq" => %{add: ~w(110)},
+    "place-aqucl" => %{add: ~w(Boat-Long Boat-Long-4 Boat-Aquarium)},
```

Comment:
`Boat-Long-4` should be `Boat-Long-South-4`.

Reply:
Updated